app/grandchallenge/ai_website/management/commands/import_data.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management import BaseCommand
+
+from grandchallenge.ai_website.utils import DataImporter
+
+
+class Command(BaseCommand):
+    help = "Reads excel database and creates sqlite database"
```

Comment:
This is only temporary, right? `xlsrd` can be removed once the initial import is done?

Reply:
The management command can actually be deleted, I have moved this functionality to a view. The import may have to be done a number of times, so I wouldn't say we can remove `xlsrd` straight away, but I think we should be able to move away from it fairly quickly.